nais.json

```diff
@@ -0,0 +1,41 @@
+{
+  "apiVersion": "nais.io/v1alpha1",
+  "kind": "Application",
+  "metadata": {
+    "name": "soknadsveiviserproxy",
+    "namespace": "default",
+    "labels": {
+      "team": "personbruker"
+    }
+  },
+  "spec": {
+    "image": "navikt/soknadsveiviserproxy:LATEST",
+    "team": "personbruker",
+    "port": 8080,
+    "ingresses": [
+      "https://soknadsveiviserproxy.nais.oera-q.local",
+      "https://tjenester-q0.nav.no/soknadsveiviserproxy"
+    ],
+    "liveness": {
+      "path": "/soknadsveiviserproxy/isAlive",
+      "initialDelay": 20
+    },
+    "readiness": {
+      "path": "/soknadsveiviserproxy/isReady",
+      "initialDelay": 20
+    },
+    "secrets": true,
```

Comment:
Jeg vil anbefale å legge til et resources-innslag her, fordi default verdiene er veldig lave. Det gjør at appen kjører veldig sakte, se gjerne på hvordan dette er tune-et for Saksoversikt: https://github.com/navikt/saksoversikt/blob/master/nais.yaml

Legg spesielt merke til resources og initialDelay, tipper at dere kan senke initialDalay hvis dere legger til konfig for resources. Dermed vil pod-ene deres starte raskere 😃